core/lib/spree/awesome_nested_set_override.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+module AwesomeNestedSetOvveride
```

Comment:
I think it may be worth adding a comment here with the contents of the commit message. This should increase the chances that, when the time is right, somebody passing by will remove this file.

Reply:
Absolutely!